go-concourse/concourse/client.go

```diff
@@ -52,3 +59,42 @@ func (client *client) URL() string {
 func (client *client) HTTPClient() *http.Client {
 	return client.connection.HTTPClient()
 }
+
+func (client *client) FindTeam(teamName string) (Team, error) {
```

Comment:
Should we move this into the team.go or teams.go file?

Reply:
Good point, forgot to move it out. In teams.go it goes!